pennylane/transforms/decompositions/single_qubit_unitary.py

```diff
@@ -156,7 +155,7 @@ def _zyz_decomposition(U, wire, return_global_phase=False):
     [RZ(12.32427531154459, wires=[0]),
      RY(1.1493817771511352, wires=[0]),
      RZ(1.733058145303424, wires=[0]),
-     (0.38469215914523336-0.9230449299422961j)*(Identity(wires=[0]))]
+     GlobalPhase((1.1759220332464762-0j), wires=[0])]
```

Comment:
I see we cast this value to complex in `_convert_to_su2`, but this seems weird to me. that value should probably at least be shown as real at the end of it all